eahub/profiles/tests/test_views.py

```diff
@@ -0,0 +1,41 @@
+from django.test import override_settings
+
+from eahub.tests.cases import EAHubTestCase
+from eahub.profiles.models import VisibilityEnum
+
+
+@override_settings(STATICFILES_STORAGE='django.contrib.staticfiles.storage.StaticFilesStorage')
```

Comment:
[optional] a good one! probably we can move this decorator to `EAHubTestCase`?

Reply:
Weirdly, ProfileTestCase does not seem to inherit the decorator from EAHubTestCase. I got a staticfiles error when I removed it and added it to EAHubTestCase instead